modules/core/src/main/java/org/apache/ignite/IgniteCache.java

```diff
@@ -1692,4 +1692,18 @@ public <T> IgniteFuture<Map<K, EntryProcessorResult<T>>> invokeAllAsync(Set<? ex
      * @return {@code True} if partition was preloaded, {@code false} if it doesn't belong to local node.
      */
     public boolean localPreloadPartition(int partition);
+
+    /**
+     * Returns number of bytes that required by the given {@code key} and corresponding value in the off-heap storage.
```

Comment:
It would be good to tell that this number is approximate: we can't estimate the exact size of unmarshalled object.

Reply:
This method provides off-heap size, so there is no `unmarshalling`.